Pilot.gyp

```diff
@@ -0,0 +1,199 @@
+      'Core/Source/MVVM/StaticViewBindingProvider.swift',
+      'Core/Source/MVVM/SwitchableModelCollection.swift',
+      'Core/Source/MVVM/View.swift',
+      'Core/Source/MVVM/ViewLayout.swift',
+      'Core/Source/MVVM/ViewModel.swift',
+      'Core/Source/MVVM/ViewModelBinding.swift',
+      'Core/Source/MVVM/ViewModelUserEvents.swift',
+      'Core/Source/Observable/Observable.swift',
+      'Core/Source/Observable/ObservableData.swift',
+      'Core/Source/Token.swift',
+      'Core/Source/Hash/SpookyV2.cpp',
+      'Core/Source/Hash/Hash.cpp',
+    ],
+    'pilot_ui_ios_sources': [
+      'UI/Source/Alerts/AlertAction.swift',
+      'UI/Source/Alerts/UIAlertController+AlertAction.swift',
+
+      'UI/Source/CollectionViews/CollectionHostedView.swift',
+      'UI/Source/CollectionViews/CollectionViewInternals.swift',
+      'UI/Source/CollectionViews/CollectionViewModelDataSource.swift',
+
+      'UI/Source/CollectionViews/ios/CollectionViewController.swift',
+      'UI/Source/CollectionViews/ios/CollectionViewHostCell.swift',
+      'UI/Source/CollectionViews/ios/CollectionViewHostReusableView.swift',
+      'UI/Source/CollectionViews/ios/Nested/NestedModelCollectionView.swift',
+
+      'UI/Source/Extensions/AssociatedObjects.swift',
+      'UI/Source/Extensions/Collection.swift',
+      'UI/Source/Layout/LayoutConstraints.swift',
+      'UI/Source/PlatformAliases.swift',
+    ],
+    'pilot_ui_mac_sources': [
+      'UI/Source/AppKitExtensions/NSCollectionView+Selection.swift',
+      'UI/Source/AppKitExtensions/NSEvent+Keys.swift',
+      'UI/Source/AppKitExtensions/NSMenu+Action.swift',
+
+      'UI/Source/CollectionViews/CollectionHostedView.swift',
+      'UI/Source/CollectionViews/CollectionViewInternals.swift',
+      'UI/Source/CollectionViews/CollectionViewModelDataSource.swift',
+
+      'UI/Source/CollectionViews/mac/CollectionView.swift',
+      'UI/Source/CollectionViews/mac/CollectionViewController.swift',
+      'UI/Source/CollectionViews/mac/CollectionViewHostItem.swift',
+      'UI/Source/CollectionViews/mac/CollectionViewHostReusableView.swift',
+
+      'UI/Source/Extensions/AssociatedObjects.swift',
+      'UI/Source/Extensions/Collection.swift',
+      'UI/Source/Layout/LayoutConstraints.swift',
+      'UI/Source/PlatformAliases.swift',
+    ],
+  },
+
+  'conditions': [
+    ['OS=="ios"',
+      {
+        'targets': [
+          {
+            'target_name': 'Pilot',
+            'mac_bundle': 1,
+            'hard_dependency': 1,
+            'type': 'static_library',
+            'mac_framework_headers': [
+              'Core/Source/Pilot.h',
+              'Core/Source/Hash/Hash.h',
+            ],
+            'mac_framework_private_headers': [
+              'Core/Source/Hash/SpookyV2.h',
+            ],
+            'sources': [
+                '<@(pilot_core_sources)',
+            ],
+            'xcode_settings': {
+              'SWIFT_VERSION': '3.2',
```

Comment:
note to self: we should probably move this out into an `xcconfig` instead (and pull it out of `Pilot.xcodeproj/project.pbxproj`).

cc @danielrhammond  - we should do this when we move to Swift 4.